src/react-apps/applications/runtime/src/styles/TextAreaComponent.css

```diff
@@ -1,3 +1,11 @@
-.textarea-disabled {
-  background-image: url("/designer/img/DesignSystem/v1/disabled-background.svg") !important;
+.disabled {
```

Comment:
This disabled class could probably be reused for some components? Maybe create a shared css file?

Reply:
fixed